database/mysql_create.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import time
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from database.config import HOST, USER, PASSWORD, PORT, DATABASE
+from database.models import Base
+from core.alert import warn
+from core.alert import messages
+
+
+def create_database():
```

Comment:
add code-based documentation for this function. specify the description and return.

Reply:
Sure, I will. This is a pending task that I will complete soon.